src/main.rs

```diff
@@ -1,12 +1,26 @@
 extern crate eno_rust;
-#[macro_use]
-extern crate runtime_fmt;
 
 use eno_rust::parser::parse;
 
 fn main() {
-    let input = "Greeting: Hello World!";
-    let locale = "en";
+    let input = "author: Jane Doe
```

Comment:
Testing strings should be moved to a separate file. This will. increase convenience of use, remove hardcodded clutter from the repo as well as will allow to gather test strings to one place (e.g. `resource` folder or smth like that)